src/SourceBrowser/src/SourceIndexServer/Program.cs

```diff
@@ -10,11 +11,19 @@ public static void Main(string[] args)
             BuildWebHost(args).Run();
         }
 
+        public static ILogger Logger { get; set; }

```

Comment:
I don't think you are using this, so we can probably remove. I don't think we are actually logging anything in the site, so something we can/should consider now that you have onboarded logging is actually adding logs that we care about and onboard into the Logging generator. No need to do in this PR of course, but perhaps something for a follow up.

Reply:
Since it took me a bit to figure out the logging flow, I was working on the basis that I would leave the scaffolding in place to be able to quickly do a bit of logging as & when in the future, rather than ripping that back out